server.go

```diff
@@ -83,6 +85,12 @@ func ServerPingInterval(interval time.Duration) ServerOption {
 	}
 }
 
+func ServerConnectivityCheckInterval(interval time.Duration) ServerOption {
```

Comment:
Given that this is, ultimately, a thift package var, does it make sense to translate it down to the client? Vs just setting the package var here?

Reply:
Good point, it certainly makes this a more concise change. I think it works because, with the exception of testing code, the `ExtensionManagerServer`  is always creating the client.